gloo/docs/installation/azure.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Azure Kubernetes Service (AKS)
+weight: 3
+---
+
+In this document we will review how to install Gloo on Azure Kubernetes Service.
+
+## Configure kubectl
+
+Configure `kubectl` to use with your cluster:
+
+```bash
+az aks get-credentials --resource-group glooResourceGroup --name glooCluster --admin
+```
+
+Replace the resource group and name with values appropriate for your environment. The `--admin` flag logs you into the cluster as the cluster administrator.
+
+Validate that `kubectl` was successfully configured with:
+
+```bash
+kubectl cluster-info
+```
+
+## Install Gloo
+
+To install Gloo, you can use one of two options:
+
+* Install via the Command Line Interface (CLI)
+* Install via Kubernetes manifest files
+
+### Install Gloo via Command Line Interface option
+
```

Comment:
There's some redundancy here with the GKE guide... I don't think it's a problem for now. May be something to clean up in the future.